src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random;
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
```

Comment:
I think you get NullPointerException here.... lets initialize random variable on 6 line